loggingpy/log.py

```diff
@@ -282,6 +288,17 @@ def format(self, record):
 
         try:
             json_dto = json.dumps(self.to_dict(dto), default=str)
+
+            if Logger.data_property_placeholder_name in dto and dto[Logger.data_property_placeholder_name] is not None:
+                if 'payload_type' in dto:
+                    property_name = dto['payload_type']
```

Comment:
I guess you made sure that the value of the `payload_type` property can not be null or an empty string? That would be a bad json field key ;)

Reply:
Both of this is fixed in the newest version of the code. It seems I pushed it right before you completed your comment.